frontend/src/common/styles/container.css

```diff
@@ -20,5 +20,6 @@
     display: flex;
     flex-direction: column;
     align-items: center;
-    background-color: #F3F3F3;
+    background-color: #ffffff;
+    box-shadow: 1rem 1rem 4rem #cbcbcb;
```

Comment:
1) extract color, shadow to a separate style
2) card-payment and other styles can inherit it https://developer.mozilla.org/en-US/docs/Web/CSS/inherit
3) Use it for auth pages as well. 
4) You can a little bit round angles like in the design